src/pipeline/resolve_import.ml

```diff
@@ -283,8 +283,13 @@ let list_files : string -> string list =
     List.filter (fun f -> Filename.extension f = ".mo") all_files
 
 let package_imports base packages =
+  let abs p = Lib.FilePath.normalise (
+    if Filename.is_relative p then Filename.concat (Sys.getcwd ()) p else p)
```

Comment:
Are you sure this should be relative to cwd? I really don't know, but if you've convinced yourself that's right, I'll go with that...